LibertyDSNP/example-client
# Bug/cannot find module 177624016

Purpose
---------------
Dependency and module location issues were preventing fresh downloads of the repository from starting correctly.

Solution
---------------
Fixed those dependency issues.


Change summary
---------------
* Fixed some dependency issues
* Updated node version
* Re-added package-lock.json back to repo.


Steps to Verify
----------------
If never downloaded:
1. pull branch
2. npm i
3. npm run watch
4. example client should start

If downloaded previously:
1. Delete modules and package-lock.json if they exist.
2. Pull branch
2. npm i
3. npm run watch
4. example client should start
